.gitignore

```diff
@@ -60,6 +60,9 @@ appengine/deploy/vault.conf
 config/ellkay.conf
 liquibase.properties
 
+# Block ALL conf files
+*.conf
+
```

Comment:
Is there any reason to not block `conf` file completely? This adds another barrier to accidental secret leaks.

Reply:
Outcome of meeting with @pegahtah  and @SimoneMaiwald: *.conf is good, also include `*.json*